tests/staking/PauseStaking.test.js

```diff
@@ -103,10 +103,10 @@ contract("Staking", (accounts) => {
         iWeightedStakingModuleMockup = await IWeightedStakingModuleMockup.at(staking.address);
 
         // Upgradable Vesting Registry
-        vestingRegistryLogic = await VestingRegistryLogic.new();
+        vestingRegistry = await VestingRegistry.new();
         vesting = await VestingRegistryProxy.new();
```

Comment:
pls replace all the occurences in the tests for better readability
```suggestion
        vestingRegistryProxy = await VestingRegistryProxy.new();
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/540/commits/4075c453d4bda8c4b6c4f16b9ca1d8f3dc4e75b9)